app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +396,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_additional_attributes_inventory(entity, hashes, target = nil)
```

Comment:
Hrm, looking at the two tables, I think @Fryguy is backwards on this.  `advanced_settings` has `default_value`, `min`, `max`, and `read_only` attributes.  Which to me looks like something that might be used to build a UI for manipulating in ManageIQ.

Whereas `custom_attributes` has just `section`, `name`, `value`, and `description`.  Which, to me means it's data we simply collect and display.

So, it looks like `custom_attributes` is where this should stay.

However, I still think it would be better to reuse the `custom_attributes` directly (so you could reuse the existing `save_custom_attributes_inventory` method, maybe with a the small change of adding `store_ids_for_new_records` to the current implementation).  Then, you could always group these Hawkular based settings with the `section` attribute of `custom_attributes`.  And, if needed, you could have a special method called `additional_attributes` that grabs these Hawkular settings.

That seems better than trying to circumvent the current inventory just to have a collection attribute new name on `container_node`.

@zeari, @zgalor, @Fryguy thoughts?